taskcluster/ci/github-release/kind.yml

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: ac_taskgraph.loader.all_dep:loader
+
+transforms:
+    - ac_taskgraph.transforms.github_release:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - post-beetmover
+
+only-for-build-types:
+    - beta
+    - release
+
+group-by: build-type
+
+job-template:
+    description: Create/Update Github release
+    worker-type: github
+    label: release-github-{build-type}-{github-project}
```

Comment:
Nit: Setting the label here shouldn't be needed anymore once we use the `all_dep` transform.

https://github.com/mozilla-mobile/android-components/blob/44987b3668e20a146b20749a8584e88d572fcfc5/taskcluster/ac_taskgraph/transforms/all_dep.py#L29